src/main/java/edu/greenblitz/robotName/commands/swerve/MoveByJoysticks.java

```diff
@@ -0,0 +1,66 @@
+package edu.greenblitz.robotName.commands.swerve;
+
+import edu.greenblitz.robotName.OI;
+import edu.greenblitz.robotName.subsystems.swerve.constants.ChassisConstants;
+import edu.greenblitz.robotName.utils.GBCommand;
+import edu.greenblitz.robotName.utils.hid.SmartJoystick;
+
+import java.util.function.DoubleSupplier;
+
+public class MoveByJoysticks extends SwerveCommand {
+
+    static double ANG_SPEED_FACTOR = 5;
+    static double LIN_SPEED_FACTOR = ChassisConstants.MAX_VELOCITY;
+    static double SLOW_ANG_SPEED_FACTOR = 0.25 * Math.PI;
+    static double SLOW_LIN_SPEED_FACTOR = 0.5;
+    private DoubleSupplier angSupplier;
+    private boolean isSlow;
+
+
+    public MoveByJoysticks(boolean isSlow, DoubleSupplier angSupplier) {
+        this.isSlow = isSlow;
+        this.angSupplier = angSupplier;
+    }
+
+    public MoveByJoysticks(boolean isSlow) {
+        this(isSlow, () -> -OI.getInstance().getMainJoystick().getAxisValue(SmartJoystick.Axis.RIGHT_X));
+    }
+    public MoveByJoysticks(boolean isSlow, boolean isFast) {
```

Comment:
Why is there need for both isSlow and isFast?

Reply:
there is no need for isFast